src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -187,6 +215,52 @@ await featureFilter.EvaluateAsync(context).ConfigureAwait(false) == targetEvalua
             return enabled;
         }
 
+        private void BindSettings(IFeatureFilterMetadata filter, FeatureFilterEvaluationContext context)
+        {
+            IFilterParametersBinder binder = filter as IFilterParametersBinder;
+
+            if (binder == null)
+            {
+                return;
+            }
+
+            object settings;
+
+            ConfigurationCacheItem cacheItem;
+
+            string cacheKey = $"{context.FeatureName}.{context.FilterIndex}";
+
+            //
+            // Check if settings already bound from configuration or the parameters have changed
+            if (!_parametersCache.TryGetValue(cacheKey, out cacheItem) ||
+                cacheItem.Parameters != context.Parameters)
+            {
+                settings = binder.BindParameters(context.Parameters);
+
+                if (_featureDefinitionProvider is IFeatureDefinitionProviderCacheable)
+                {
+                    _parametersCache.Set(
+                        cacheKey,
+                        new ConfigurationCacheItem
+                        {
+                            Settings = settings,
+                            Parameters = context.Parameters
+                        },
+                        new MemoryCacheEntryOptions
+                        {
+                            SlidingExpiration = TimeSpan.FromMinutes(5),
+                            AbsoluteExpirationRelativeToNow = TimeSpan.FromDays(1)
```

Comment:
Can we define these constants somewhere instead of hardcoding inline?

Reply:
This Wrapper class allows us to update the "ref" when configurations are updated- regardless if the configuration updated it's reference or not. I believe that's it's only purpose- unless Jimmy had other reasons in mind.